text/0003-multidimensional-values-representation.md

```diff
@@ -0,0 +1,27 @@
+* Start date: 2018-03-23
+* Contributors: Andrei Korigodski <akorigod@gmail.com>
+* RFC PR: [mavlink/rfcs#3](https://github.com/mavlink/rfcs/pull/3)
+
+# Summary
+
+Agree on the representation of the common multidimensional quantities.
+
+# Motivation
+
+In different messages common multidimensional quantities are defined in different ways: sometimes using an array and sometimes using separate fields for every dimension (for example, `x`, `y`, `z` for positions or `w`, `x`, `y`, `z` for quaternions).
```

Comment:
NOTE: usually quaternions is `float[4]`, but in older messages it's often `q1, q2, q3, q4`.

Reply:
Thanks! Fixed.